CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
+- [Security] Updated Brave Wallet password policy. ([#6093](https://github.com/brave/brave-ios/issues/6093))
+- Implemented "Bridge to Aurora" for Brave Wallet. ([#5785](https://github.com/brave/brave-ios/issues/5785))
+- Updated Portfolio settings page in Brave Wallet. ([#8089](https://github.com/brave/brave-ios/issues/8089))
+- Updated button to create wallet to be disabled when wallet is already being created. ([#8357](https://github.com/brave/brave-ios/issues/8357))
+- Updated “Send” screen in Brave Wallet to not show decimal values for zero balances. ([#8232](https://github.com/brave/brave-ios/issues/8232))
+- Updated “Save Changes” button to be enabled only when there is a change in wallet portfolio settings. ([#8229](https://github.com/brave/brave-ios/issues/8229))
+- Fixed “view my recovery phrase” not displaying in Brave Wallet. ([#8445](https://github.com/brave/brave-ios/issues/8445))
+
+### General 
+
+- Added support for all history data to be synced when "History" sync is enabled. ([#8305](https://github.com/brave/brave-ios/issues/8305))
+- Added setting to ensure YouTube links always load in brave. ([#8272](https://github.com/brave/brave-ios/issues/8272))
+- Added confirmation message for successfully exporting bookmarks. ([#8236](https://github.com/brave/brave-ios/issues/8236))
+- Added confirmation message for resetting configuration in VPN settings. ([#8087](https://github.com/brave/brave-ios/issues/8087))
+- Added support for additional schemas for going to the App Store. ([#8035](https://github.com/brave/brave-ios/issues/8035))
+- Added VPN improvements for callouts and the settings page. ([#4879](https://github.com/brave/brave-ios/issues/4879))
+- Added the ability to download .mobileconfig files and auto-import into settings app. ([#4358](https://github.com/brave/brave-ios/issues/4358))
+- Implemented tabs bar re-order via drag and drop. ([#8058](https://github.com/brave/brave-ios/issues/8058))
+- Removed SDWebImage SVG coder. ([#7026](https://github.com/brave/brave-ios/issues/7026))
+- Removed cookie consent info popup. ([#8217](https://github.com/brave/brave-ios/issues/8217))
+- Removed the dislike swipe action for ad notifications. ([#8211](https://github.com/brave/brave-ios/issues/8211))
+- Updated the “Send” button in “Send to your Devices” window to be displayed in the header section. ([#8329](https://github.com/brave/brave-ios/issues/8329))
+- Updated filter lists to increase the amount of engines that can be enabled on low memory devices. ([#8292](https://github.com/brave/brave-ios/issues/8292))
+- Updated the “Report a Broken Site” submission form to include details and contact fields. ([#8270](https://github.com/brave/brave-ios/issues/8270))
+- Updated user agent for iOS 17. ([#8155](https://github.com/brave/brave-ios/issues/8155))
+- Updated browser colors. ([#8078](https://github.com/brave/brave-ios/issues/8078))
```

Comment:
Pretty vague. I'm no 100% sure what to put but can probably improve the above. Maybe something like the following?

```
Updated browser colors for "Light", "Dark" and "Private" mode in several areas including "Reader Mode", "Tab Switcher", "Favorites & Search" and several other areas.
```

Reply:
I like that